aten/src/ATen/core/Error.cpp

```diff
@@ -5,15 +5,17 @@
 #include <numeric>
 #include <string>
 
+using std::string;
```

Comment:
...why. This seems strictly worse

Reply:
Also, like, if you're actually going to do this, do it in a separate diff.

Reply:
It's in a .cpp file, so `using` statements help conciseness/readability and namespaces aren't polluted.

Reply:
We don't have working lint rules to prevent people from putting using declarations like this in headers, so it seems easier for code reviewers to just forbid this globally.

Reply:
wait, so we're shunning `using` entirely?

Reply:
It's definitely not encouraged